e2e/rbd.go

```diff
@@ -1945,10 +1944,9 @@ var _ = Describe("RBD", func() {
 						}
 					}()
 
-					var pvc = &v1.PersistentVolumeClaim{}
-					pvc, err = loadPVC(pvcPath)
-					if err != nil {
-						e2elog.Failf("failed to load PVC with error %v", err)
+					pvc, pvcErr := loadPVC(pvcPath)
```

Comment:
I'm surprised the change of naming `err` is suggested. It seems this is the only location where that is done?

Reply:
No No. Actually the static-check was failing for this code, as the `err` here was overlapping the one declared above.